### PR TITLE
add man page in mdoc

### PR DESCRIPTION
**Pull request by epsilon-0**
does not handle the man page installation.

### CODE REVIEW

File: wayland-logout.1 [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.Dd $Mdocdate: November 27 2020 $
+.Dt WAYLAND-LOGOUT 1
+.Os
+.Sh NAME
+.Nm wayland-logout
+.Nd generic logout command for wayland compositors
```
Comment (soreau) [inline wayland-logout.1]:
```suggestion
.Nd generic logout utility for wayland compositors
```

File: wayland-logout.1 [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+.Sh DESCRIPTION
+.Nm
+is a compositor agnostic command for simulating a
```
Comment (soreau) [inline wayland-logout.1]:
```suggestion
is a compositor agnostic utility for simulating a
```

File: wayland-logout.1 [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+logout with any wayland complaint compositor.
```
Comment (soreau) [inline wayland-logout.1]:
```suggestion
logout with any wayland compositor.
```

File: wayland-logout.1 [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+.Pp
+It sends a
+.Sy SIGINT
+.Xr signal 3
+to the running compositor that is obtained by looking
```
Comment (soreau) [inline wayland-logout.1]:
```suggestion
to the running compositor pid, which is obtained by looking
```

File: wayland-logout.1 [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+variables.
+.Sh ENVIRONMENT
+The command uses the
+.Ev WAYLAND_DISPLAY
+and
+.Ev XDG_RUNTIME_DIR
+.Nm
+environment variables to get the
+.Xr socket 2
+corresponding to the compositor instance.
```
Comment (soreau) [inline wayland-logout.1:28]:
```suggestion
corresponding to the compositor instance. .Sy SIGINT is sent to the pid owning the socket.
```

File: wayland-logout.1 [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+.Sh EXIT STATUS
+The
+.Nm
+command exits with 1 if an error has occurred along with
```
Comment (soreau) [inline wayland-logout.1]:
```suggestion
utility exits with 1 if an error has occurred along with
```

File: wayland-logout.1 [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+the error reasoning and exits with 0 on success.
+.Sh EXAMPLES
+To kill the current compositor instance, run inside
+the compositor:
+.Dl $ wayland-logout
+.Pp
+A typical use case, to kill a specific compositor
+instance of another user (with uid=1002), run as
+the other user (or as root):
```
Comment (soreau) [inline wayland-logout.1:40]:
```suggestion
another user:
```

File: wayland-logout.1 [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+.Dl $ env -i XDG_RUNTIME_DIR=/run/user/1002 WAYLAND_DISPLAY=wayland-4 wayland-logout
```
Comment (soreau) [inline wayland-logout.1:41]:
```suggestion
.Dl $ env -i XDG_RUNTIME_DIR=/run/user/1002 WAYLAND_DISPLAY=wayland-4 wayland-logout
or
.Dl $ WAYLAND_DISPLAY=/run/user/1002/wayland-4 wayland-logout
```

File: wayland-logout.1 [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+.Xr socket 2
+corresponding to the compositor instance.
+.Sh EXIT STATUS
+The
+.Nm
+command exits with 1 if an error has occurred along with
+the error reasoning and exits with 0 on success.
+.Sh EXAMPLES
+To kill the current compositor instance, run inside
+the compositor:
```
Comment (soreau) [inline wayland-logout.1]:
```suggestion
the compositor environment:
```

**Review by soreau — CHANGES_REQUESTED** [review]
Looks really good from the start, just a few minor nitpicks.

File: wayland-logout.1 (hunk 1 above)
Comment (epsilon-0) [reply wayland-logout.1]: done

File: wayland-logout.1 (hunk 2 above)
Comment (epsilon-0) [reply wayland-logout.1]: done

File: wayland-logout.1 (hunk 4 above)
Comment (epsilon-0) [reply wayland-logout.1]: done

File: wayland-logout.1 (hunk 6 above)
Comment (epsilon-0) [reply wayland-logout.1]: done

File: wayland-logout.1 (hunk 7 above)
Comment (epsilon-0) [reply wayland-logout.1:40]: hmm, this doesn't flow right. 
We already referred to the user with uid=1002, it doesn't make sense to call another user again.

File: wayland-logout.1 (hunk 8 above)
Comment (epsilon-0) [reply wayland-logout.1:41]: done

File: wayland-logout.1 (hunk 9 above)
Comment (epsilon-0) [reply wayland-logout.1]: done

File: wayland-logout.1 (hunk 5 above)
Comment (epsilon-0) [reply wayland-logout.1:28]: This is the environment variables section.
We already referred to the `SIGINT` in the previous section.
Seems a bit superfluous.

File: wayland-logout.1 (hunk 3 above)
Comment (epsilon-0) [reply wayland-logout.1]: done

File: wayland-logout.1 [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+which is obtained by looking at the
+.Sx environment
+variables.
+.Sh ENVIRONMENT
+.Nm
+uses the
+.Ev WAYLAND_DISPLAY
+and
+.Ev XDG_RUNTIME_DIR
+.Nm
```
Comment (soreau) [inline wayland-logout.1]:
```suggestion
```

Comment (epsilon-0) [reply wayland-logout.1]: done.

**Review by soreau — APPROVED** [review]

**Comment by soreau** [comment]
Thanks!

**Pull request merged**